itou/templates/search/includes/siaes_search_form.html

```diff
@@ -1,7 +1,7 @@
 {% load bootstrap4 %}
 
 {% bootstrap_form_errors form type="all" %}
-<div class="row no-gutters my-3{% if align == 'center' %} justify-content-sm-center{% endif %}">
+<div class="row no-gutters {% if align == 'center' %} justify-content-sm-center{% endif %}">
```

Comment:
Je fait le commentaire ici et avec cette correction, mais l'idée principale c'est de choisir 1 forme et d'harmoniser avec `prescribers_search_form.html` :).
```suggestion
<div class="row no-gutters{% if align == 'center' %} justify-content-sm-center{% endif %}">
```